raiden/tests/utils/blockchain.py

```diff
@@ -47,6 +47,8 @@ def wait_until_block(chain, block):
     curr_block = chain.block_number()
     while curr_block < block:
         curr_block = chain.next_block()
+    # ctx switch to allow background tasks to proceed:
+    gevent.sleep(1)
```

Comment:
The `chain.block_number`  does context-switches, I don't think we need this sleep.

If we keep the line the sleep time should be a smaller, e.g. `gevent.sleep(0.000001)`.